### PR TITLE
Ascii art command option

### LINKED ISSUE #2: Add command eval support to ASCII Text Art component
Currently the text art component just displays the machine's hostname, which is obviously not the only thing to turn into cool text art. Should allow a configurable command to run and turn into text art.

### PR DESCRIPTION
**Pull request by filalex77**
Closes #2

### CODE REVIEW

File: lib/panda_motd/default_config.yaml [hunk 1]
```diff
@@ -16,12 +16,14 @@ components:
   #     can also use 'default' for your default terminal color. Additionally,
   #     you can prefix any of the base colors with 'light_' to get the lighter
   #     version.
+  #   command: The command to generate the text. Defaults to hostname.
```
Comment (taylorthurlow) [inline lib/panda_motd/default_config.yaml]: Saying "defaults to hostname" I think might be a bit misleading in this case. If you completely left out the "command" configuration then the script crashes. I think we should remove the text here.

That said, this does bring up the fact that basically none of these values (for any component, for that matter) have true defaults. They'd all break the script if any of them were left. Maybe that's a good thing, maybe not.

Comment (filalex77) [reply lib/panda_motd/default_config.yaml]: You think this comment line should be removed altogether?

**Comment by taylorthurlow** [comment]
Ah sorry maybe I wasn't clear. The line can stay I just don't think we should say "defaults to hostname" at the end.

**Comment by filalex77** [comment]
@taylorthurlow It's fine. You'll probably squash it anyways :)

**Pull request merged**